widgetssdk/src/main/java/com/glia/widgets/messagecenter/MessageCenterActivity.kt

```diff
@@ -27,6 +27,27 @@ import com.glia.widgets.helper.fixCapturedPhotoRotation
 import com.glia.widgets.helper.mapUriToFileAttachment
 import java.io.IOException
 
+/**
+ * This activity is used for displaying secure conversations welcome screen.
+ *
+ * Main features:
+ * - Returns an error if visitor is not authenticated or specified queue does not support secure conversations
+ * - Provides possibility to send async messages
+ * - Provides possibility to go to chat history
+ *
+ * Before this activity is launched, make sure that Glia Widgets SDK is set up correctly.
+ *
+ * Required data that should be passed together with the Activity intent:
+ * - {@link GliaWidgets#QUEUE_ID}: ID of the queue which you would like to use for your engagement
+ * For full list of optional parameters refer to constants defined in {@link GliaWidgets}
+ *
+ * Code example:
+ * ```
```

Comment:
Just to understand if this is something that I need to include in the style guide, should we format code examples with `<pre>` tags in Java and with ` ``` ` in Kotlin?